src/ds/d500/d500-auto-calibration.cpp

```diff
@@ -432,6 +432,12 @@ namespace librealsense
     void d500_auto_calibrated::write_calibration() const
     {
         _calib_engine->write_calibration(_curr_calibration);
+
+        for( auto & cb : _depth_write_callbacks )
```

Comment:
The write does it inside no?

Reply:
Did not understand. `_calib_engine->write_calibration()` uses a HWMC to write the data to the camera.
The callbacks are to notify users - in this case the device object